app/src/main/java/org/cru/godtools/ui/dashboard/tools/CategoryAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package org.cru.godtools.ui.dashboard.tools
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.ObservableField
+import androidx.lifecycle.Observer
+import org.ccci.gto.android.common.recyclerview.adapter.SimpleDataBindingAdapter
+import org.cru.godtools.databinding.DashboardListItemCategoryBinding
+
+class CategoryAdapter() :
+    SimpleDataBindingAdapter<DashboardListItemCategoryBinding>(), Observer<List<String>> {
+
+    val callbacks = ObservableField<CategoryAdapterCallbacks>()
+    var selectedCategory: String? = null
+        set(value) {
+            field = value
+            notifyDataSetChanged()
+        }
```

Comment:
you should just provide the selected category LiveData directly instead of duplicating it's state to this variable and having to re-bind all categories when the selection changes